sycl/source/detail/scheduler/commands.cpp

```diff
@@ -1397,13 +1397,22 @@ void MemCpyCommand::printDot(std::ostream &Stream) const {
   }
 }
 
-AllocaCommandBase *ExecCGCommand::getAllocaForReq(Requirement *Req) {
+AllocaCommandBase *ExecCGCommand::getAllocaForReqOrThrow(Requirement *Req) {
+  auto allocaCmd = getAllocaForReq(Req);
```

Comment:
```suggestion
  auto AllocaCmd = getAllocaForReq(Req);
```

Note: I'm not entirely sure if we have a requirement for RT code for local variables to be in `CamelCase`. But the surrounding code seem to follow that code style

Reply:
> Why there are not test changes in this PR? There should either be a new test for a missing case this PR fixes or a change to enable previously disabled test (due to missing case)

The new tests are in this [PR](https://github.com/intel/llvm-test-suite/pull/1549/) in `llvm-test-suite`.